packages/sdk/app-framework/src/plugins/PluginHost/plugin.ts

```diff
@@ -66,9 +66,14 @@ export type Plugin<TProvides = {}> = {
 
     /**
      * Component to render icon for the plugin when displayed in a list.
+     * @deprecated
      */
-    // TODO(wittjosiah): Convert to `icon` and make serializable.
     iconComponent?: FC<IconProps>;
+
+    /**
+     * A grep-able symbol string which can be resolved to an icon asset by @ch-ui/icons, via @ch-ui/vite-plugin-icons.
```

Comment:
```suggestion
     * A symbol string which can be resolved to an icon which represents the plugin.
```
I don't think the app framework actually knows about ch-ui nor does it need to. Honestly I'd consider just renaming this to `icon` on the next pass of this api.